### PR TITLE
feat: add @opentelemetry/plugins-all-node-contrib

### PR DESCRIPTION
**Pull request by dyladan**
This is related to #20

### CODE REVIEW

File: metapackages/plugins-all-node-contrib/package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/open-telemetry/opentelemetry-js-contrib.git"
+  },
+  "bugs": {
+    "url": "https://github.com/open-telemetry/opentelemetry-js-contrib/issues"
+  },
+  "dependencies": {
+    "@opentelemetry/plugin-dns": "^0.7.0",
```
Comment (dyladan) [inline metapackages/plugins-all-node-contrib/package.json:17]: Remove for now until context based tracing disable is implemented

**Comment by dyladan** [comment]
Add a README

**Comment by andrewhsu** [comment]
@dyladan thanks for this! as discussed in the SIG mtg today, adding a README or documentation with this to reserve the right to update it to make sure stuff is added (or not added if it causes problems) would be good.

**Comment by dyladan** [comment]
Closing in favor of only having `plugins-core` and `plugins-all`. The use case to only install contrib plugins probably is not that valuable and increases maintenance burden.

**Pull request closed without merging**